src/NRedisStack/NRedisStackConfigurationOptions.cs

```diff
@@ -0,0 +1,36 @@
+using NRedisStack.RedisStackCommands;
+using StackExchange.Redis;
+
+namespace NRedisStack
+{
+    public class NRedisStackConfigurationOptions
+    {
+        private readonly ConfigurationOptions _configurationOptions = new ConfigurationOptions();
+
+        public NRedisStackConfigurationOptions(string redisConnectionString)
+        {
```

Comment:
Remove this constructor, add a `Parse` method that pulls out one of these from a Redis URI or an  StackExchange.Redis Connection String. I'd just assume that if the connection string isn't a valid URI that you should try parsing a `ConfigurationOptions` from it.